docs/arcadia-users-group/20220906-intro-to-shell1/lesson.md

```diff
@@ -0,0 +1,294 @@
+This is as opposed to controlling your computer using a graphical user interface (GUI) with a mouse/keyboard/touchscreen combination.
+
+Why should we care about learning the shell?
+
+* Many bioinformatics tools can only be used through a command-line interface, and/or have extra capabilities in the command-line version that are not available in the GUI (e.g. BLAST).  
+* It makes our work less error-prone. When humans do the same thing a hundred different times (or even ten times), we're likely to make a mistake. Your computer can do the same thing a thousand times with no mistakes. This also frees us up to do other things we can't automate, like the science part.  
+* The shell makes our work more reproducible. When we carry out our work in a command-line interface (rather than a GUI), we can keep an exact record of all steps, which we can use to re-do our work when we need to. It also gives us a way to unambiguously communicate what we've done to others.  
+* Many bioinformatic tasks require large amounts of computing power and can't realistically be run on our own machines. These tasks are best performed using remote computers or cloud computing, which can only be accessed through a shell.
+
+## Common shell terminology
+
+Many different terms are used to refer to the shell or its computing environment. 
+Some commons ones are included below with their definitions.
+
+| **Term**     | **What it is**  |
+|:------------:|:---------------:|
+| **`shell`** | what we use to talk to the computer; anything where you are pointing and clicking with a mouse is a **G**raphical **U**ser **I**nterface (**GUI**) shell; something with text only is a **C**ommand **L**ine **I**nterface (**CLI**) shell | 
+| **`command line`** | a text-based environment capable of taking input and providing output (a "terminal" is the same idea) |
+| **`Unix`** | a family of operating systems |
+| **`bash`** | the most common programming language used at a Unix command-line |
+
+
+## Accessing the shell for this lesson
+
+For this lesson, we need to have access to a Unix shell. 
+Click the button below to launch a shell through [binder](https://mybinder.org/).
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/arcadia-science/arcadia-computational-training/main)
+
+This will launch a computer in the cloud.
+You'll interact with this computer through your browser.
+Click the Terminal button to launch a Terminal that we will work with for the rest of the lesson.
+
+![](jupyterhub.png)
+
+<details>
+  <summary>More information on binder and what happens when you click the launch binder button.</summary>
+
+binder is a service that turns a Git repo into a collection of interactive notebooks. 
+When a repository is configured to run as a binder, build a docker image that contains all of the specified software installations.
+binder launches an "instance" in the cloud (either on Google Cloud or AWS typically) with the docker image.
```

Comment:
What is a Docker image? If I haven't specified a Dockerfile in my repo, how does Binder build a Docker image?